tools/downloader/pytorch_to_onnx.py

```diff
@@ -22,6 +22,21 @@ def positive_int_arg(values):
     return result
 
 
+def model_parameters(parameters):
+    if not parameters:
+        return dict()
+    params = {}
+    for element in parameters.split(','):
+        param, value = element.split('=')
```

Comment:
You should use the `maxsplit` parameter to prevent splitting on more than one equals sign.